### ISSUE TITLE
[Windows][Vulkan] Weird visual glitches produced by move animations, potentially related somehow to resizing Yuzu's window (Pokémon Brilliant Diamond v1.1.1)

### ISSUE THREAD

**Issue by LOuroboros** [yuzu-emu/yuzu#7377]
**Version of Yuzu**: EA 2218.
**Details**: I was looking for a Chansey, and I noticed that from one moment to another, its move's animations were executed in a visually glitchy manner.

![yuzu_2083](https://user-images.githubusercontent.com/4485172/142650977-3baa03fd-10d8-4e55-831f-999273d34584.png)

https://user-images.githubusercontent.com/4485172/142651620-a33d9a41-235a-442d-8d3c-e5e8ed16cda4.mp4

Here's the log:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7571346/yuzu_log.txt)
***Note**: I'm not sure why does it say Windows 10 Version 2009, but to clarify, I have Windows 10 21H1 x64, Build 19043.1165.*

**EDIT**: Oh yeah, there's a couple of details I forgot.
When I said "potentially related somehow to resizing Yuzu's window", that's because I believe I changed the window from maximized windowed mode to just regular window mode.
Interestingly, the glitches were partially fixed when I swapped out my Murkrow. Particle effects such as the stat reductions worked normally, unlike in the video where they appear as small and white dots, but the issue where the battle background turned into a colorful mess was still present.

**Comment by FernandoS27** [comment]
Please provide a save.

**Comment by LOuroboros** [comment]
> Please provide a save.

Here you go.
[0100000011D90000.zip](https://github.com/yuzu-emu/yuzu/files/7571669/0100000011D90000.zip)

**Comment by FernandoS27** [comment]
> > Please provide a save.
> 
> Here you go. [0100000011D90000.zip](https://github.com/yuzu-emu/yuzu/files/7571669/0100000011D90000.zip)

could ypu tell me how to reproduce it?

**Comment by LOuroboros** [comment]
> > > Please provide a save.
> > 
> > 
> > Here you go. [0100000011D90000.zip](https://github.com/yuzu-emu/yuzu/files/7571669/0100000011D90000.zip)
> 
> could ypu tell me how to reproduce it?

I'm not sure. As I said, I just got into a battle and when Chansey used one of their moves, the animation was executed in a glitchy manner. From then on, all their move animations also suffered similar problems.

I may or may not have resized Yuzu's window from maximized window to just a regular window, I don't remember. If I did, it was rather unconsciously or without thinking it too much.

**Comment by FernandoS27** [comment]
> > > > Please provide a save.
> > > 
> > > 
> > > Here you go. [0100000011D90000.zip](https://github.com/yuzu-emu/yuzu/files/7571669/0100000011D90000.zip)
> > 
> > 
> > could ypu tell me how to reproduce it?
> 
> I'm not sure. As I said, I just got into a battle and when Chansey used one of their moves, the animation was executed in a glitchy manner. From then on, all their move animations also suffered similar problems.
> 
> I may or may not have resized Yuzu's window from maximized window to just a regular window, I don't remember. If I did, it was rather unconsciously or without thinking it too much.

could you capture a chansey for me and give me a save?

**Comment by LOuroboros** [comment]
> > > > > Please provide a save.
> > > > 
> > > > 
> > > > Here you go. [0100000011D90000.zip](https://github.com/yuzu-emu/yuzu/files/7571669/0100000011D90000.zip)
> > > 
> > > 
> > > could ypu tell me how to reproduce it?
> > 
> > 
> > I'm not sure. As I said, I just got into a battle and when Chansey used one of their moves, the animation was executed in a glitchy manner. From then on, all their move animations also suffered similar problems.
> > I may or may not have resized Yuzu's window from maximized window to just a regular window, I don't remember. If I did, it was rather unconsciously or without thinking it too much.
> 
> could you capture a chansey for me and give me a save?

There's like 7 or 8 of them inside the PC. You can go to a Pokémon Center and take one of them out.

If you can't, let me know and I will.

**Comment by FernandoS27** [comment]
It's now fixed

**Comment by LOuroboros** [comment]
> It's now fixed

I'm not sure if I should open a new issue or not @FernandoS27, but the problem where particle effects do not play properly at times *(0:37 in my video above, when Chansey uses Tail Whip)* is still there.

I booted up the game in maximized window mode, then when I got InGame I turned it into a regular window and just went through some encounters. Randomly, from one battle to another *(I defeated a Mime Jr. and then I walked around to trigger another encounter afterward)*, the particles of certain move animations *(most notably stat de/buffs)*
 stopped being casted on the screen correctly.

https://user-images.githubusercontent.com/4485172/142730633-2488a9ed-f877-4efd-b438-b3da252ee997.mp4

For reference, this is how a stat debuff animation should look like, see this wild Staravia's Growl:

https://user-images.githubusercontent.com/4485172/142730992-dd7e83d8-d380-4eba-bf02-d66f02eb63c2.mp4

Log:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7574694/yuzu_log.txt)
